packages/custom-field/src/vaadin-custom-field-mixin.js

```diff
@@ -303,6 +304,10 @@ export const CustomFieldMixin = (superClass) =>
 
     /** @private */
     __applyInputsValue(value) {
+      if (value == null) {
+        return;
+      }
```

Comment:
Shouldn't passing `null` clear the value for all sub fields?

Reply:
Yes, probably. Maybe we should then use `\t` (or default value) when `null` is passed.